data/phones/list_phones.py

```diff
@@ -97,17 +95,16 @@ def main(args: argparse.Namespace):
     ):
         print(
             f"{phone:<5}"
-            " # "
+            "\t# "
             f"{len(examples):<10}"
```

Comment:
I was using the formatting like `:<5` and `:<10` to achieve what a tab character would do but with a bit more control. We should probably do one or the other, but not both?

Reply:
The `<5` thing isn't working for me (it seems to be counting in Unicode codepoints or bytes, not in number of spaces), hence the tab. But the `<10` bit is still wise IMO. See next commit.

Reply:
Makes sense -- LGTM.